src/main/java/com/openlattice/hazelcast/serializers/EntitySetStreamSerializer.java

```diff
@@ -69,6 +67,12 @@ public static void serialize( ObjectDataOutput out, EntitySet object ) throws IO
 
         out.writeInt( object.getPartitionsVersion() );
 
+        if ( object.getExpiration() != null ) {
+            out.writeBoolean( true );
+            DataExpirationStreamSerializer.serialize( out, object.getExpiration() );
+        } else {
+            out.writeBoolean( false );
+        }
```

Comment:
Can use `KotlinOptionalStreamSerializers.serializeMaybeValue` here

Reply:
oooooh fancy new thing?

Reply:
eehhh fancy, not-so-much. New, yes!

Reply:
Oof nvm, the semantics aren't great if invoked from Java 😢

Reply:
bummer :(